src/style/mod.rs

```diff
@@ -257,7 +274,7 @@ pub struct TextBoxStyle<C, F, A, V, H> {
     pub line_spacing: i32,
```

Comment:
Is this parameter the spacing between the bounding boxes of two adjacent lines or the difference in the baseline position? I did plan to add a `line_height: u32` parameter to `TextStyle` and I think we should use the same in e-g and e-t.

Reply:
Right now it's the space between 2 bounding boxes, but probably should be the other one.